src/Build/BackEnd/Client/MSBuildClient.cs

```diff
@@ -195,10 +200,74 @@ public MSBuildClientExitResult Execute(CancellationToken cancellationToken)
             _numConsoleWritePackets = 0;
             _sizeOfConsoleWritePackets = 0;
 
+            ReadPacketsLoop(cancellationToken);
+
+            MSBuildEventSource.Log.MSBuildServerBuildStop(descriptiveCommandLine, _numConsoleWritePackets, _sizeOfConsoleWritePackets, _exitResult.MSBuildClientExitType.ToString(), _exitResult.MSBuildAppExitTypeString);
```

Comment:
Why do you need both ExitType.ToString() and ExitTypeString? If we need the former, can we wrap this in MSBuildEventSource.Log.EventsEnabled or whatever it is?

Reply:
Since this is called one per build, this would have negligible perf gains.